### ISSUE TITLE
Question regarding elapsedTimestamp

### ISSUE THREAD

**Issue by jvanattedev** [mauriciolauffer/openui5-error-collector#22]
Do I understand this correctly: this is the number of milliseconds since the initial pageload?

**Comment by mauriciolauffer** [comment]
Yes, that's correct. It uses the `event.timeStamp` property: https://developer.mozilla.org/en-US/docs/Web/API/Event/timeStamp

**Comment by jvanattedev** [comment]
Does it refer to another moment than this
* @property {string} timestamp - When event was logged

**Comment by mauriciolauffer** [comment]
Yes, it does. Have a look at the demo: https://mauriciolauffer.github.io/openui5-error-collector/demo/index.html

- [timestamp](https://github.com/mauriciolauffer/openui5-error-collector/blob/be26367b75762d4e128e90ee012d02a8d1bd5ca4/src/ui5ErrorCollector.js#L10) - When event was logged
- [elapsedTimestamp](https://github.com/mauriciolauffer/openui5-error-collector/blob/be26367b75762d4e128e90ee012d02a8d1bd5ca4/src/ui5ErrorCollector.js#L16) - Elapsed time since page load

<img width="264" alt="image" src="https://github.com/mauriciolauffer/openui5-error-collector/assets/443888/3265a246-4828-4b83-954d-b61e533467ca">
